pagopa/io-app
# chore: new discovery screen

## Short description
This PR refactors the IT Wallet discovery screen used for the wallet activation.

## List of changes proposed in this pull request
- Refactored `ItwDiscoveryInfoScreen` to adapt it to the new design
- Updated `ItwMarkdown` component
  - Added correct spacing in `ItwMarkdown` component
  - Moved `ItwMarkdown` component into common/components folder
  - Added `ItwMarkdown`  snapshot testing
- Added `ItwSmallText` component
  - Added `ItwSmallText` component to render the ToS text starting from a markdown string
  - Added `ItwSmallText` snapshot testing
- Added required locales
- Added required assets

## How to test
From the **IT Wallet playground**, navigate to the **Discovery** feature and check that the component is rendered correctly.

## Preview

https://github.com/pagopa/io-app/assets/6160324/c34a06bb-edd5-49b9-a65a-c8f38ee6e760
